clang/lib/CIR/Dialect/Transforms/TargetLowering/LowerFunction.cpp

```diff
@@ -79,6 +79,64 @@ Value enterStructPointerForCoercedAccess(Value SrcPtr, StructType SrcSTy,
                  // above.
 }
 
+// CoerceIntOrPtrToIntOrPtr - Convert a value Val to the specific Ty where both
```

Comment:
Please remove the `CoerceIntOrPtrToIntOrPtr -` part!

Reply:
removed, fixed `///` style